tests/acceptance/pageObjects/filesPage.js

```diff
@@ -101,6 +100,50 @@ module.exports = {
         .useXpath()
         .waitForElementNotPresent('@loadingIndicator')
         .waitForElementNotPresent(this.getFileRowSelectorByFileName(element))
+    },
+    toggleFilterFileOrFolder: async function (fileOrFolder, enableOrDisable) {
+      const filterSelector = {}
+      if (fileOrFolder === 'folder') {
+        filterSelector['filterLabel'] = '@filterFolderLabel'
+        filterSelector['filterCheckbox'] = '@filterFolderCheckbox'
+        filterSelector['filterCheckboxId'] = this.elements['filterFolderCheckbox']
+      } else if (fileOrFolder === 'file') {
+        filterSelector['filterLabel'] = '@filterFileLabel'
+        filterSelector['filterCheckbox'] = '@filterFileCheckbox'
+        filterSelector['filterCheckboxId'] = this.elements['filterFileCheckbox']
+      } else {
+        throw new Error(`Expected 'file' or 'folder', ${fileOrFolder} given`)
+      }
+      await this
+        .waitForElementVisible('@filterListButton')
+        .click('@filterListButton')
+        .waitForElementVisible(filterSelector.filterLabel)
+
+      return this
+        .toogleCheckbox(
+          enableOrDisable,
+          filterSelector.filterCheckbox,
+          filterSelector.filterCheckboxId
+        )
+    },
+    toogleCheckbox: function (enableOrDisable, filterCheckbox, filterCheckboxId) {
```

Comment:
maybe we have to place it in customCommands see http://nightwatchjs.org/guide#writing-custom-commands